analyses/tmb-calculation/run_tmb_calculation.sh

```diff
@@ -12,35 +12,35 @@ script_directory="$(perl -e 'use File::Basename;
 cd "$script_directory" || exit
 
 # Set up paths to data files consumed by analysis
-data_path="../../data"
-scratch_path="../../scratch"
-bed_files_path="input"
+data_dir="../../data"
+scratch_dir="../../scratch"
+input_dir="input"
 
 # BED and GTF file paths
-cds_file="${scratch_path}/gencode.v27.primary_assembly.annotation.bed"
-wgs_bed="${scratch_path}/intersect_strelka_mutect2_vardict_WGS.bed"
+cds_file="${scratch_dir}/gencode.v27.primary_assembly.annotation.bed"
+wgs_bed="${scratch_dir}/intersect_strelka_mutect2_vardict_WGS.bed"
```

Comment:
@ewafula - could you please remind me which module generate these two files and put them in scratch? I think for reproducibility, we might want to move those files either to `references` folder for this module or add them to data release.

Reply:
@runjin326, they are created by the same script (`run_tmb_calculation.sh`); just a few lines of code below.  They are written to `scratch/` because they are intermediate temporary files created every time the module is run but never uploaded to the GitHub repo when the PR is submitted. 

https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/895f0858468f75354c36db45ab5bd0adfbd2c212/analyses/tmb-calculation/run_tmb_calculation.sh#L33-L48

Reply:
Ah - gotcha! That makes sense. then everything looks good to me!